sup.rb

```diff
@@ -0,0 +1,41 @@
+require "language/go"
```

Comment:
Can we move this file under `release/` directory? One day, when we reach v1.0, we should probably try to open PR against https://github.com/Homebrew/homebrew/tree/master/Library/Formula